### PR TITLE
FLU: Obstacles/toys/panels/ball/drinks

### PR DESCRIPTION
**Pull request by BenjaminDeleze**
- [x] (298) objects/obstacles/protos/Ramp30deg.proto
- [x] (299) objects/obstacles/protos/OilBarrel.proto
- [x] (300) objects/obstacles/protos/ThreeSteps.proto
- [x] (301) objects/toys/protos/PaperBoat.proto
- [x] (302) objects/toys/protos/RubberDuck.proto
- [x] (303) objects/panels/protos/PanelWithTubes.proto
- [x] (304) objects/panels/protos/PannelWithTubes.proto (nothing to do)
- [x] (305) objects/panels/protos/Panel.proto
- [x] (308) objects/balls/protos/RobocupSoccerBall.proto
- [x] (309) objects/balls/protos/FifaSoccerBall.proto (nothing to do)
- [x] (310) objects/balls/protos/Ball.proto (nothing to do)
- [x] (311) objects/balls/protos/PingPongBall.proto (nothing to do)
- [x] (312) objects/balls/protos/SoccerBall.proto (nothing to do)
- [x] (313) objects/advertising_board/protos/AdvertisingBoard.proto
- [x] (314) objects/drinks/protos/BeerBottle.proto
- [x] (315) objects/drinks/protos/WaterBottle.proto
- [x] (316) objects/drinks/protos/Can.proto

### CODE REVIEW

File: projects/objects/toys/protos/PaperBoat.proto [hunk 1]
```diff
@@ -51,53 +51,53 @@ PROTO PaperBoat [
...
             Capsule {
               height 0.342
               radius 0.014
             }
           ]
         }
-        Transform {
-          translation 0 0.038 0.0308
-          rotation 0.27209603 -0.27209511 0.92299946 1.6508407
+      Transform {
```
Comment (lukicdarkoo) [inline projects/objects/toys/protos/PaperBoat.proto:71]: This looks wrong

File: docs/guide/object-obstacles.md [hunk 2]
```diff
@@ -81,10 +81,10 @@ Derived from [Solid](../reference/solid.md).
 
 ```
 ThreeSteps {
-  SFVec3f    translation  0 -0.5 0
+  SFVec3f    translation  0 0 -0.5
```
Comment (lukicdarkoo) [inline docs/guide/object-obstacles.md:84]: This is properly converted, but it looks strange having half of an object sank into the ground

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]

File: docs/guide/object-obstacles.md (hunk 2 above)
Comment (BenjaminDeleze) [reply docs/guide/object-obstacles.md:84]: Yes, I think it is to allow an `increment` of 1

Comment (lukicdarkoo) [reply docs/guide/object-obstacles.md:84]: Interesting :)

**Review by lukicdarkoo — APPROVED** [review]
Thank you

**Review by lukicdarkoo — APPROVED** [review]

**Pull request merged**